uiowahjmjohnsonselt2024/projectdirectory-selt_2024_team_009
# Pull Request:Push Recent Code Updates for Wallet Functionality to Heroku

# Title: 

## What does this PR do?
This pull request includes the recent code updates from Madhu (IMPL: added wallet functionality, new member bonus, buying shards #123). Additionally, minor changes were made for compatibility with Heroku to ensure smooth deployment and functionality.

## Related Issues:
- Issue #123: Implement wallet functionality, new member bonus, and shard purchasing

## Testing:
1. Deploy the updated code to Heroku (`young-sierra-77487` app).
2. Test the wallet functionality to ensure proper behavior on Heroku.
3. Verify the new member bonus logic works as intended.
4. Confirm shard purchasing functionality operates correctly.
5. Run all existing test cases to ensure no regressions.

## Checklist:
- [x] Deployment to Heroku completed successfully
- [x] All features (wallet, bonus, shards) tested and verified
